media/model/commandrouter.js

```diff
@@ -2,13 +2,18 @@ const crypto = require('crypto');
 const VideoHandler = require('./media/videohandler');
 const ImageHandler = require('./media/imagehandler');
 const WebsiteHandler = require('./media/websitehandler');
+const AudioHandler = require('./media/audiohandler');
 
 module.exports = class CommandRouter {
 
     constructor(dom, fileHandler) {
         this.componentId =
           process.env.SCREENCRASH_COMPONENT_ID ||
           crypto.randomBytes(8).toString('hex');
+        this.supportedTypes = null;
```

Comment:
Could this be initialized to ['video', 'image', 'web', 'audio'] instead? Then we wouldn't need to handle a magic null value.